### ISSUE TITLE
have a bug in ubuntu

### ISSUE THREAD

**Issue by cjx-dev** [tauri-apps/tauri-action#440]
![image](https://user-images.githubusercontent.com/64702233/233666840-bceda47c-760b-4ac9-a2b9-1ab7fa573e92.png)
_Error failed to rename `/home/runner/work/slep-test/slep-test/target/release/nextjs-slep` to `/home/runner/work/slep-test/slep-test/target/release`: Directory not empty (os error 39)_
but in macos or windows it's successful
```
name: Release

on:
  push:
    tags:
      - 'v*'
  workflow_dispatch:

jobs:
  release:
    permissions:
      contents: write
    strategy:
      fail-fast: false
      matrix:
        platform: [macos-latest, ubuntu-20.04, windows-latest]
    runs-on: ${{ matrix.platform }}

    defaults:
      run:
        working-directory: ./nextjs-frontend

    steps:
      - name: Checkout repository
        uses: actions/checkout@v3

      - name: Setup pnpm
        uses: pnpm/action-setup@v2
        with:
          version: 7.1.7

      - name: Install Protoc
        uses: arduino/setup-protoc@v1
        with:
          repo-token: ${{ secrets.GITHUB_TOKEN }}

      - name: Install dependencies (ubuntu only)
        if: matrix.platform == 'ubuntu-20.04'
        # You can remove libayatana-appindicator3-dev if you don't use the system tray feature.
        run: |
          sudo apt-get update
          sudo apt-get install -y libgtk-3-dev libwebkit2gtk-4.0-dev libayatana-appindicator3-dev librsvg2-dev

      - name: Rust setup
        uses: dtolnay/rust-toolchain@nightly

      - name: Rust cache
        uses: swatinem/rust-cache@v2
        with:
          workspaces: './src-tauri -> target'

      - name: Sync node version and setup cache
        uses: actions/setup-node@v3
        with:
          node-version: 'lts/*'
          cache: 'pnpm' # Set this to npm, yarn or pnpm.
          cache-dependency-path: nextjs-frontend/pnpm-lock.yaml

      - name: Install frontend dependencies
        # If you don't have `beforeBuildCommand` configured you may want to build your frontend here too.
        run: pnpm install

      - name: Build the app
        uses: tauri-apps/tauri-action@v0
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          tagName: ${{ github.ref_name }} # This only works if your workflow triggers on new tags.
          releaseName: 'App Name v__VERSION__' # tauri-action replaces \_\_VERSION\_\_ with the app version.
          releaseBody: 'See the assets to download and install this version.'
          releaseDraft: true
          prerelease: false

```

**Comment by FabianLars** [comment]
Would it be possible for you to share the project?

If not, what did you set productName in tauri.conf.json to? What's the `name` field in Cargo.toml and package.json set to?

**Comment by FabianLars** [comment]
See https://github.com/tauri-apps/tauri/issues/5860. Should be fixed once you upgrade `@tauri-apps/cli` to v1.3.0
